clients/src/test/java/org/apache/kafka/common/requests/ApiVersionsResponseTest.java

```diff
@@ -160,25 +161,30 @@ public void shouldNotCreateApiResponseWithTelemetryWhenDisabled() {
             build();
         verifyApiKeysForTelemetry(response, 0);
     }
-
+    
     @Test
-    public void testMetadataQuorumApisAreDisabled() {
+    public void testBrokerApisAreEnabled() {
         ApiVersionsResponse response = new ApiVersionsResponse.Builder().
             setThrottleTimeMs(AbstractResponse.DEFAULT_THROTTLE_TIME).
             setApiVersions(ApiVersionsResponse.filterApis(
-                ListenerType.ZK_BROKER,
+                ListenerType.BROKER,
                 true,
                 true)).
             setSupportedFeatures(Features.emptySupportedFeatures()).
             setFinalizedFeatures(Collections.emptyMap()).
             setFinalizedFeaturesEpoch(ApiVersionsResponse.UNKNOWN_FINALIZED_FEATURES_EPOCH).
             build();
-        // Ensure that APIs needed for the KRaft mode are not exposed through ApiVersions until we are ready for them
-        HashSet<ApiKeys> exposedApis = apiKeysInResponse(response);
-        assertFalse(exposedApis.contains(ApiKeys.VOTE));
-        assertFalse(exposedApis.contains(ApiKeys.BEGIN_QUORUM_EPOCH));
-        assertFalse(exposedApis.contains(ApiKeys.END_QUORUM_EPOCH));
-        assertFalse(exposedApis.contains(ApiKeys.DESCRIBE_QUORUM));
+
+        Set<ApiKeys> exposed = apiKeysInResponse(response);
+
+
+        Arrays.stream(ApiKeys.values())
+            .filter(key -> key.messageType.listeners().contains(ListenerType.BROKER))
+            .forEach(key -> assertTrue(exposed.contains(key)));
+        Arrays.stream(ApiKeys.values())
+            .filter(key -> key.messageType.listeners()
+                .stream().noneMatch(listener -> listener == ListenerType.BROKER))
```

Comment:
Nit: I think we could have done `!contains` instead of `stream.noneMatch`, but I think it's fine as it is.